src/mantine-dropzone/src/Dropzone.tsx

```diff
@@ -110,6 +124,8 @@ export const defaultProps: Partial<DropzoneProps> = {
   dragEventsBubbling: true,
   activateOnKeyboard: true,
   useFsAccessApi: true,
+  getFilesFromEvent: fromEvent,
```

Comment:
There is no need to specify default prop for this

Reply:
I must set a default value to these props otherwise they are undefined values ​​and the behavior will not be as expected.

Although I could do the following in the useDropzone params:

  ```
const { getRootProps, ...otherValues } = useDropzone({
    ...otherParams
    getFilesFromEvent: getFilesFromEvent || fromEvent,
    validator: validator || null,
  });
```

Reply:
useDropzone hook has its own default values. There is no need to provide default value from our side. Another unwanted side effect is that `@mantine/dropzone` package will have to depend on `file-selector` to make it work correctly with yarn v3.